appcontroller.yaml

```diff
@@ -6,9 +6,9 @@ spec:
   restartPolicy: Always
   containers:
   - name: kubeac
-    image: gluke77/k8s-ac
-    command:  ["/usr/bin/run_runit"]
+    image: nebril/k8s-appcontroller
+    command:  ["sh", "run_runit.sh"]
```

Comment:
run_runit.sh is executable, it invokes /bin/sh itself, there is no need to call extra sh